generated/nidigital/nidigital/session.py

```diff
@@ -2940,30 +2940,46 @@ def write_source_waveform_site_unique(self, waveform_name, waveform_data):
         self.sites[site_list]._write_source_waveform_site_unique_u32(waveform_name, len(waveform_data), actual_samples_per_waveform, data)
 
     @ivi_synchronized
-    def get_channel_name_from_string(self, index):
-        r'''get_channel_name_from_string
+    def get_channel_names(self, indices):
+        r'''get_channel_names
 
-        TBD
+        Returns a list of channel names for given channel indices.
```

Comment:
Where did this documentation come from? It won't be obvious to NI-Digital users why this is useful. The answer is that it's useful in multi-instrument sessions. The same is true for NI-SCOPE.

For NI-DCPower users there are more scenarios where this will be useful.

I think we should mention it.